tests/data/test_withdrawal.cairo

```diff
@@ -59,14 +60,12 @@ fn test_set_withdrawal_new_and_override() {
     let (caller_address, role_store, data_store) = setup();
     let account = 'account'.try_into().unwrap();
     // TODO make these Span32
-    let long_token_swap_path: Span<ContractAddress> = array![
-        1.try_into().unwrap(), 2.try_into().unwrap(), 3.try_into().unwrap()
-    ]
-        .span();
-    let short_token_swap_path: Span<ContractAddress> = array![
-        4.try_into().unwrap(), 5.try_into().unwrap(), 6.try_into().unwrap()
-    ]
-        .span();
+    let long_token_swap_path: Span32<ContractAddress> = Array32Trait::span32(
```

Comment:
it doesnt work with `array![
        1.try_into().unwrap(), 2.try_into().unwrap(), 3.try_into().unwrap()].span32()`?